blueocean-events/src/test/java/io/jenkins/blueocean/events/SseEventTest.java

```diff
@@ -505,24 +526,32 @@ public OrganizationImpl of(ItemGroup group) {
         }
     }
 
-    private void isEquals(String source, String target, AtomicInteger errorCount){
-        if(!source.equals(target)){
-            System.err.println(String.format("Source: %s is not equals to target: %s", source, target));
-            errorCount.incrementAndGet();
+    private static class AssertionHelper {
```

Comment:
What's the point of all this, if it calls `fail()`? If calling it doesn't end the test by throwing, can't we just rely on the test runner to count the errors during summation? If it does, why not just use normal asserts? I think I must be missing something here.

Reply:
because, in async block unit assertions or fail doesn't fail the test. Reason I am calling fail is, so that looking at log we can find which test and where it failed.